index.html

```diff
@@ -1,12 +1,262 @@
 <!DOCTYPE html>
 <html lang="ru">
+
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
+    <title>Гостиницы Седоны</title>
+    <link rel="stylesheet" href="styles.css">
   </head>
+
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
 
+    <!-- Шапка -->
+
+
+    <header class="page-header" data-test="header">
+      <nav class="page-navigation">
+        <ul class="navigation-list">
+
+          <!-- Logo -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="index.html" aria-current="page">
+              <img class="page-header-logo" src="images/same/logo.svg" width="140" height="70"
+                alt="Логотип сайта города Седоны.">
+            </a>
+          </li>
+
+          <!-- Главная -->
+          <li class="navigation-item">
+            <a class="navigation-link navigation-link-current" href="page.html">Главная</a>
+          </li>
+
+          <!-- О Седоне -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="about_Sedona.html">О Седоне</a>
+          </li>
+
+          <!-- Гостиницы -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="hotel.html">Гостиницы</a>
+          </li>
+
+        </ul>
+        <ul class="navigation-list navigation-user">
+
+          <!-- Поиск -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Поиск</span>
+            </a>
+          </li>
+
+          <!-- В избранном -->
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">В избранном</span>
+              <span class="user-vaforites">12</span>
+            </a>
+          </li>
+
+          <!-- ХОЧУ СЮДА! -->
+          <li class="navigation-item">
+            <a class="button navigation-link" href="#">ХОЧУ СЮДА!</a>
+          </li>
```

Comment:
```suggestion
          <li class="navigation-item">
            <a class="button navigation-link" href="#">Хочу сюда!</a>
          </li>
```
Текст в коде мы всегда должны писать по правилам языка страницы. 
По-русски фразы пишутся с большой буквы и потом маленькие. 
Если по дизайну нужно, чтобы все были заглавными, то меняем это стилями и text-transform.